### PR TITLE
Implemented Cross-Origin-Resource-Policy response header

### PR DESCRIPTION
**Pull request by miguelcrpinto**
Implemented  Cross-Origin-Resource-Policy response header according to the description on issue #76

### CODE REVIEW

**Comment by GaProgMan** [comment]
Thank you for this. I'll take a look at it today.

File: src/Models/CrossOriginResourcePolicy.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        public const string SameOriginValue = "same-origin";
+        /// <summary>
+        /// Only requests from the same Site can read the resource.
+        /// </summary>
+        public const string SameSiteValue = "same-site";
+        /// <summary>
+        /// Requests from any Origin (both same-site and cross-site) can read the resource. 
+        /// Browsers are using this policy when an CORP header is not specified.
+        /// </summary>
+        public const string CrossOriginValue = "same-origin";
```
[the remark this answers is not in the document]
Comment (miguelcrpinto) [reply src/Models/CrossOriginResourcePolicy.cs]: It was indeed a copy-paste issue! Fixed!

**Comment by miguelcrpinto** [comment]
@jamie-taylor-rjj  I also bumped the version and updated the changelog accordingly. Can you please check if I changed everything needed?

**Comment by jamie-taylor-rjj** [comment]
In case you were wondering, I was using [this guide](https://github.com/erikthedeveloper/code-review-emoji-guide) when choosing emoji for the PR comments. Looking at this new version of the PR now, live on Twitch.

**Comment by jamie-taylor-rjj** [comment]
closes #76

**Comment by miguelcrpinto** [comment]
Thanks! I managed to still watch a few minutes of the review and merge on twitch!

**Pull request merged**